docs/_docs/user-guide/eldritch.md

```diff
@@ -692,6 +692,11 @@ The random library is designed to enable generation of cryptogrphically secure r
 
 The <b>random.bool</b> method returns a randomly sourced boolean value.
 
+### random.string
+
+`random.string(length: uint) -> str`
```

Comment:
Can you add an optional parameter to define the characters the function can draw from.

You can use the Option<Vec<char>>
To achieve this.
An example of the option parameter in use can found in the process.info function.

https://github.com/spellshift/realm/blob/a0904a79267f0e919ea53343c3cb5d3a1b2789ce/implants/lib/eldritch/src/process/info_impl.rs#L11

Reply:
Or maybe a String instead of Vec<char> might be more ergonomic

Reply:
Also please add the default character set it will use.

Reply:
Option added and documentation updated 😄